packages/amplify-graphql-api-construct/package.json

```
@@ -209,8 +209,8 @@
     "@aws-sdk/nested-clients": "^3.777.0",
     "@aws-sdk/region-config-resolver": "3.614.0",
     "@aws-sdk/token-providers": "3.614.0",
-    "@aws-sdk/types": "3.609.0",
-    "@aws-sdk/util-endpoints": "3.637.0",
+    "@aws-sdk/types": "3.734.0",
```

```suggestion
    "@aws-sdk/types": "^3.734.0",
```

So that we don't have to do it again in the future. 
this should be ok for types package.